[2024-06-27T04:40:16.968+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-27T04:40:16.988+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: demo_dag_for_alerting_apis_final.check_previous_run_status scheduled__2024-06-27T04:39:00+00:00 [queued]>
[2024-06-27T04:40:16.996+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: demo_dag_for_alerting_apis_final.check_previous_run_status scheduled__2024-06-27T04:39:00+00:00 [queued]>
[2024-06-27T04:40:16.996+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 6
[2024-06-27T04:40:17.011+0000] {taskinstance.py:2330} INFO - Executing <Task(BranchPythonOperator): check_previous_run_status> on 2024-06-27 04:39:00+00:00
[2024-06-27T04:40:17.015+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=214) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-06-27T04:40:17.016+0000] {standard_task_runner.py:63} INFO - Started process 215 to run task
[2024-06-27T04:40:17.017+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'demo_dag_for_alerting_apis_final', 'check_previous_run_status', 'scheduled__2024-06-27T04:39:00+00:00', '--job-id', '4530', '--raw', '--subdir', 'DAGS_FOLDER/demo_dag_for_alerting_apis.py', '--cfg-path', '/tmp/tmpgjs5c87d']
[2024-06-27T04:40:17.017+0000] {standard_task_runner.py:91} INFO - Job 4530: Subtask check_previous_run_status
[2024-06-27T04:40:17.057+0000] {task_command.py:426} INFO - Running <TaskInstance: demo_dag_for_alerting_apis_final.check_previous_run_status scheduled__2024-06-27T04:39:00+00:00 [running]> on host eeacb89bcfc1
[2024-06-27T04:40:17.150+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='demo_dag_for_alerting_apis_final' AIRFLOW_CTX_TASK_ID='check_previous_run_status' AIRFLOW_CTX_EXECUTION_DATE='2024-06-27T04:39:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-06-27T04:39:00+00:00'
[2024-06-27T04:40:17.151+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-27T04:40:17.164+0000] {baseoperator.py:399} WARNING - BranchPythonOperator.execute cannot be called outside TaskInstance!
[2024-06-27T04:40:17.164+0000] {python.py:237} INFO - Done. Returned value was: start_run
[2024-06-27T04:40:17.164+0000] {branch.py:36} INFO - Branch into start_run
[2024-06-27T04:40:17.164+0000] {skipmixin.py:178} INFO - Following branch start_run
[2024-06-27T04:40:17.177+0000] {skipmixin.py:246} INFO - Skipping tasks [('skip_run', -1)]
[2024-06-27T04:40:17.199+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-27T04:40:17.217+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=demo_dag_for_alerting_apis_final, task_id=check_previous_run_status, run_id=scheduled__2024-06-27T04:39:00+00:00, execution_date=20240627T043900, start_date=20240627T044016, end_date=20240627T044017
[2024-06-27T04:40:17.270+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-27T04:40:17.341+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-06-27T04:40:17.345+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
